src/components/PreviewHint.js

```diff
@@ -26,6 +26,7 @@ function PreviewHint({ roles, suggestion }) {
     <Expandable
       excerpt={excerpt}
       className="bg-gray-200 text-gray-800 font-mono text-xs rounded fle"
+      data-testid="preview"
```

Comment:
Instead of using the testid we could use `section` element for `exandable` and add an `aria-label` to it?
Like this schema. What do you think?
![image](https://user-images.githubusercontent.com/5365582/101328165-f6ea2f00-386f-11eb-9866-73306aee3142.png)

cc @smeijer

Reply:
@marcosvega91 as far as I can tell, `section` is not selectable by Testing Library, with or without `aria-label`. Your comment made me realize that the `data-testid` here is superfluous, as `Expandable` doesn't appear to be inheriting arbitrary props and applying them to any of its elements. The test I wrote doesn't actually use the `testid` here, so I'll remove it.

I was intentional about not changing too much of the markup or the internals of components just for the sake of writing tests, as I figured suggestions for refactoring to make the page more accessible/testable would be a good conversation among contributors.

Reply:
My idea was to make these parts more testable, but we can do this in a future PR without any problem :) . 

The section element [has a region role](https://testing-playground.com/gist/b01b0ea70bbe66825dd5559089e43f66/1626de0b634e46784fbace3cf77bf46ccb645705).

Reply:
Oh, yeah, thanks for pointing out the `region` role.

I've updated `Expandable` to render a `section` and to accept a `labelText` property to set the `aria-label`. The resulting tests are definitely cleaner this way. Let me know what you think.